lib/mongo/cluster/topology.rb

```diff
@@ -35,14 +29,26 @@ class Cluster
     # @since 2.0.0
     module Topology
       extend self
+    end
+  end
+end
 
+require 'mongo/cluster/topology/replica_set_no_primary'
```

Comment:
What's the purpose of moving the require statements to the middle of the file?

Reply:
I'd move them to the bottom but that does not work.

Reply:
Sounds good! Thanks for explaining